README.md

```diff
@@ -255,6 +259,110 @@ Here is an example:
 }
 ```
 
+#### SPECTER specific parameters (affinity scores):
```

Comment:
Should we assume these three new models need to be run using the virtulenv `specter` specified here: https://github.com/allenai/specter#how-to-use-the-pretrained-model?

Reply:
I'll add the steps explicitly to the README since we want it installed in the same environment as openreview-expertize